internal/constants/constants.go

```diff
@@ -53,3 +54,51 @@ const (
 	// UUIDSize is the length of a UUID, in bytes.
 	UUIDSize = 16
 )
+
+// domain migration flags
+const (
```

Comment:
https://golang.org/pkg/net/#Flags

How about adding `type MigrateFlags uint32`, and then the below constants use that type and become `MigrateFlagsLive`, `MigrateFlagsPeerToPeer`, etc.